### PR TITLE
fix irrigation task css

### PR DESCRIPTION
**Pull request by wasswarichard**
Fixes elements  CSS styling and remove click event on overlay  https://lite-farm.atlassian.net/browse/LF-2800

### CODE REVIEW

**Comment by anant00729** [comment]
This looks good to me. @Duncan-Brain or @kcussen Please review.

**Review by anant00729 — APPROVED** [review]

File: packages/webapp/src/components/Task/PureIrrigationTask/index.jsx [hunk 1]
```diff
@@ -107,6 +109,10 @@ export default function PureIrrigationTask({
   const onDismissWaterUseCalculatorModel = () => setShowWaterUseCalculatorModal(false);
   const handleModalSubmit = () => {
     setValue(ESTIMATED_WATER_USAGE, totalWaterUsage);
+    setValue(ESTIMATED_WATER_USAGE_UNIT, {
+      label: 'l',
+      value: 'l',
+    });
```
[the remark this answers is not in the document]
Comment (JimmyRowland) [reply packages/webapp/src/components/Task/PureIrrigationTask/index.jsx:115]: getUnitOptionMap() should probably be typed

[the remark this answers is not in the document]
Comment (wasswarichard) [reply packages/webapp/src/components/Task/PureIrrigationTask/index.jsx:115]: @JimmyRowland I don't have that function getUnitOptionMap().

File: packages/webapp/src/components/Task/PureIrrigationTask/index.jsx [hunk 2]
```diff
@@ -132,9 +138,7 @@ export default function PureIrrigationTask({
         tooltipContent={
           <>
             {t('ADD_TASK.IRRIGATION_VIEW.BRAND_TOOLTIP.FIRST_PHRASE')}{' '}
```
[the remark this answers is not in the document]
Comment (wasswarichard) [reply packages/webapp/src/components/Task/PureIrrigationTask/index.jsx:140]: @JimmyRowland that's a concatenation of strings.

**Comment by Duncan-Brain** [comment]
@wasswarichard I see the css and tooltip link being fixed here. Those look fine. 

What's the other stuff though? If it is for the water use calculator LF-2801 maybe make a new PR because there are other calculator problems.

**Comment by wasswarichard** [comment]
@Duncan-Brain all those changes are part of the ticket. Check out the ticket for the suggested enhancements https://lite-farm.atlassian.net/browse/LF-2800

File: packages/webapp/src/components/Task/PureIrrigationTask/index.jsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (JimmyRowland) [reply packages/webapp/src/components/Task/PureIrrigationTask/index.jsx:115]: https://github.com/LiteFarmOrg/LiteFarm/blob/a144be6fc8f6789a50c1fc2a025ae05911ab31cc/packages/webapp/src/components/Form/Unit/index.jsx#L37

**Comment by Duncan-Brain** [comment]
@wasswarichard Looks good.

Sorry about my misunderstanding. I couldn't see what it was you were talking about on LF-2800 before. I see it now that the three things you are handling here are:

- the top margin between the “Estimated water usage” and “Not sure?” should be 4px
- Remove the link for “field work task” in the overlay
- Values in the estimated water usage are not updating when the unit changes (which is inconsistent with LF-2801)

**Review by Duncan-Brain — APPROVED** [review]

**Pull request merged**